### PR TITLE
Document how inspection of packets traces in EDTT works

### PR DESCRIPTION
**Pull request by Tronil**
Packet inspection in EDTT (using BSIM) was never documented; Added some basic information on what it is and how to use it

Signed-off-by: Troels Nilsson <trnn@demant.com>

### CODE REVIEW

File: docs/EDTT_framework_Babblesim.md [hunk 1]
```diff
@@ -110,6 +110,64 @@ All requests and replies share a common syntax of the form:
...
+* `data`: The raw data of the packet (excluding header)
+* `type`: The type of packet, for example `'ADV_IND'` or `'CONNECT_IND'`
+* `header`: The header of the packet - a named tuple containing `pdu_type`, `ch_sel`, `tx_add`, `rx_add` and `payload_len`
+* `payload`: The decoded payload of the packet. Content depends on the type of packet; It will generally be a named tuple containing the fields specified in the BT Core Spec. One notable exception is for the extended advertisement packets - for these the payload is a `dict` since the fields may or may not be present
+
+### Packets class interface
+
+There are a few different ways to inquire about packets sent on the phy from within a test case. Generally, the way to use the API is to filter for the packets you are interested - see the API describtion below.
+
+#### Packets.fetch(packet_filter=())
```
Comment (aescolar) [inline docs/EDTT_framework_Babblesim.md:134]: Does calling fetch get the set of packets updated? (meaning it goes to the files to look for more?)

**Review by aescolar — COMMENTED** [review]
Thanks, no complaints from me. Just a question, as I have not looked into this code :)

File: docs/EDTT_framework_Babblesim.md [hunk 2]
```diff
@@ -110,6 +110,64 @@ All requests and replies share a common syntax of the form:
...
 ```
 
+## Packet inspection
+
+It is possible to inspect the raw packets sent by the upper and lower tester via BabbleSims dump files. Note that you will usuallly want to use the `-dump_imm` command line argument when using packet inspection; Otherwise the dump files will use cached writes and the latest packets will likely not show up in EDTT.
+
+The raw packets will get decoded into a `Packet` before being handed over to the test cases via the `Packets` class. The content of a `Packet` is:
+
+* `direction`: Either `'Tx'` or `'Rx'`. Note that the current implementation only reads the Tx part of the dump files, so this will currently always be `'Tx'`
+* `idx`: The BSim device id of the device that received or transmitted the package (see `direction` for whether it was a transmit or receive). Will correesppnd to lower or upper tester
```
Comment (wopu-ot) [inline docs/EDTT_framework_Babblesim.md:120]: `correesppnd` -> `correspond`

**Review by wopu-ot — CHANGES_REQUESTED** [review]
Looks good, just a typo to fix.

File: docs/EDTT_framework_Babblesim.md (hunk 1 above)
Comment (Tronil) [reply docs/EDTT_framework_Babblesim.md:134]: Yes it reads automagically from the files :-)

File: docs/EDTT_framework_Babblesim.md (hunk 2 above)
Comment (Tronil) [reply docs/EDTT_framework_Babblesim.md]: Fixed

**Review by wopu-ot — APPROVED** [review]

File: docs/EDTT_framework_Babblesim.md (hunk 1 above)
Comment (aescolar) [reply docs/EDTT_framework_Babblesim.md:134]: Ok, it would be nice to mention it in the doc :)

**Pull request merged**